File: doc/snippets/rcs/capabilities/snippet.js
const url = 'https://api.smsbox.pro/rcs/json/<AGENTID>/capabilities';
const headers = {
    'Authorization': 'App <APIKEY>',
    'Content-Type': 'application/json'
};
const data = {
    recipients: ["+336XXXXXXXX"]
};

fetch(url, {
    method: 'POST',
    headers: headers,
    body: JSON.stringify(data)
})
    .then(response => response.json())
    .then(data => console.log(data))
    .catch(error => console.error('Error:', error));

File: doc/snippets/rcs/capabilities/snippet.py
import requests
import json

url = 'https://api.smsbox.pro/rcs/json/<AGENTID>/capabilities'
headers = {
    'Authorization': 'App <APIKEY>',
    'Content-Type': 'application/json'
}
data = {
    "recipients": ["+336XXXXXXXX"],
}

response = requests.post(url, headers=headers, data=json.dumps(data))

print(response.status_code)
print(response.json())
